src/controller/HomeController.php

```diff
@@ -38,10 +42,10 @@ private function __construct()
      * 
      * @return \App\controller\HomeController
      */
-    public static function getInstance() :HomeController
+    public static function getInstance(TemplateInterface $_templateEngine) :HomeController
     { 
         if (is_null(self::$_instance)) {
-            self::$_instance = new HomeController();  
+            self::$_instance = new HomeController($_templateEngine);  
```

Comment:
```suggestion
            self::$_instance = new HomeController($templateEngine);  
```

On fait référence ici à la variable de ton paramètre de ta fonction

Reply:
ok